### DISCUSSION TITLE
Including renovate in CI/CD per project, instead of centrally?

### DISCUSSION THREAD

**Discussion by Yannik** [renovatebot/renovate#30032] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Gitlab, latest

### Please tell us more about your question or problem

Since renovate does not support using multiple access tokens, we have to run multiple docker containers.

Thus, I thought of not creating a separate project for renovate, but configuring renovate directly in the `.gitlab-ci.yml` of each individual project that should use renovate. Is this a recommended approach? 

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
It's more work to set up (one pipeline per project), and then you'll need to monitor them somehow, but otherwise there is no "don't do this" recommendation from us. I'd recommend you consider one semi-central pipeline per *group* still.
